README.md

```diff
@@ -15,17 +15,6 @@ We are currently piloting this project with a small number of news organizations
 2. [Architecture](#architecture)
 3. [What's In This Repo?](#whats-in-this-repo)
 2. [Installation](#installation)
-   - [Install Qubes](#install-qubes)
-   - [Development Environment](#development-environment)
-   - [Staging Environment](#staging-environment)
-3. [Development](#development)
-   - [Testing](#testing)
-   - [Automatic Updates](#automatic-updates)
-   - [Building the Templates](#building-the-templates)
-   - [Building workstation deb packages](#building-workstation-deb-packages)
-   - [Building workstation rpm packages](#building-workstation-deb-packages)
-4. [Using the *SecureDrop Client*](#using-the-securedrop-client)
-5. [Distributing and Releasing](#distributing-and-releasing-workstation-and-its-subprojects)
 6. [Threat model](#threat-model)
```

Comment:
Good catch! The smallest nit: Numbering is now off further. ;-) (Markdown doesn't care, which is why the weird 2. 3. 2. numbering above is ignored in the rendered view, but I'd suggest fixing both while we're here.)

Reply:
Sure, I'll fix it :+1:

Reply:
Done!